src/Microsoft.QualityGuidelines.Analyzers/Core/PreferJaggedArraysOverMultidimensional.cs

```diff
@@ -21,38 +24,131 @@ public abstract class PreferJaggedArraysOverMultidimensionalAnalyzer : Diagnosti
         private static readonly LocalizableString s_localizableMessageBody = new LocalizableResourceString(nameof(MicrosoftQualityGuidelinesAnalyzersResources.PreferJaggedArraysOverMultidimensionalMessageBody), MicrosoftQualityGuidelinesAnalyzersResources.ResourceManager, typeof(MicrosoftQualityGuidelinesAnalyzersResources));
         private static readonly LocalizableString s_localizableDescription = new LocalizableResourceString(nameof(MicrosoftQualityGuidelinesAnalyzersResources.PreferJaggedArraysOverMultidimensionalDescription), MicrosoftQualityGuidelinesAnalyzersResources.ResourceManager, typeof(MicrosoftQualityGuidelinesAnalyzersResources));
 
+        private const string helpLink = "https://msdn.microsoft.com/en-us/library/ms182277.aspx";
+
         internal static DiagnosticDescriptor DefaultRule = new DiagnosticDescriptor(RuleId,
```

Comment:
Yes the {0} thingie means one can't use LocalizableStrings which is bad. We probably just need an analyzer that makes sure that when descriptors are different they only differ in the messageFormat.